docs/js-lang-overview.md

```diff
@@ -57,9 +57,11 @@ The target SQL data type defines how the system converts values. The system typi
 
 JS `null` and `undefined` values always convert to SQL `NULL` values for the target SQL type.
 
+Here's the fixed raw markdown table:
+
 | Target SQL data type | Rules for converting JS value | Example |
 |----------------------|----------------------------|---------|
-| Integer Numbers (BOOLEAN, TINYINT, SHORTINT, MEDIUMINT, INT, BIGINT) | JS Integers/Numbers: integers stored as-is, BIGINTs attempted as integers, others as strings. | 42 → 42 <br>  3.14 → "3.14" <br> true → "true"|
+| Integer Numbers (BOOLEAN, TINYINT, SHORTINT, MEDIUMINT, INT, BIGINT) | (Version 8.4.5) Integer Number values are stored in the SQL parameter as integer.<br>Boolean values are converted to `0/1` and stored in the SQL parameter as integer.<br>TINYINT, SHORTINT, MEDIUMINT, INT values are stored in the SQL parameter as integer.<br>If possible, BIGINT values are stored in the SQL parameter as integer.<br>All other JS values, including non-integer numbers, are converted to strings, and the result is stored in the SQL parameter.<br>(Version 8.4.4 - JS Integers/Numbers: integers stored as-is, BIGINTs attempted as integers, others as strings.) | 42 → 42<br>3.14 → "3.14"<br>true → "0" |
 | DECIMAL | Converted to strings | 123.45 → "123.45" |
```

Comment:
I think it is worth mentioning special handling of JS Boolean values for DECIMAL and FLOAT/DOUBLE types.